civil/civil.go

```diff
@@ -86,6 +86,17 @@ func (d Date) AddDays(n int) Date {
 	return DateOf(d.In(time.UTC).AddDate(0, 0, n))
 }
 
+// AddDate returns the time corresponding to adding the given number of years,
```

Comment:
Nit: feel free to do in your own commit
```suggestion
// AddDate returns the date corresponding to adding the given number of years,
```